oshi-core/pom.xml

```diff
@@ -47,6 +47,11 @@
 			<version>4.4.0</version>
 		</dependency>
 		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
```

Comment:
Have you looked into any documentation requirements for including an Alv2 dependency into an EPL product?